bin/test-workflow/etc/secretless.yml

```diff
@@ -0,0 +1,34 @@
+version: "2"
```

Comment:
I think this file can be deleted. It's only used for the secretless-broker integration, AND, when we add that integration, we'll probably add this as data contained in a ConfigMap, and that ConfigMap will get deployed by Helm install.

Reply:
Deleted